kili/cli/project/member/add.py

```diff
@@ -0,0 +1,145 @@
+            project_id=project_id, email=email, disable_tqdm=True)
+        if (len(existing_user) > 0 and existing_user[0]['activated']):
+            warnings.warn(f'{email} is already an active member of the project.'
+                          ' Use kili project member update to update role.')
+        else:
+            kili.append_to_roles(project_id=project_id,
+                                 user_email=email, role=role)
+            count += 1
+    return count
+
+
+@click.command()
+@Options.api_key
+@Options.endpoint
+@click.argument('inputs', type=str, required=True, nargs=-1)
+@ click.option('--project-id', type=str, required=True,
+               help='Id of the project to add members to')
+@ click.option('--role', type=str, default=None,
+               show_default='LABELER',
+               help='Project role of the added user(s).')
+# pylint: disable=too-many-branches
+def add_member(api_key: Optional[str],
+               endpoint: Optional[str],
+               inputs: str,
+               project_id: str,
+               role: Optional[str],
+               ):
+    """Add members to a Kili project
+
+    Arguments can be: \n
+        - string (with email format) \n
+        - path to a csv file with email in the first column \n
+            + optional: role in the second column \n
+        - a project_id of another kili project \n
+
+    If the argument is the project_id of another Kili project,
+    copy the user from this project (and their role).
+
+    You need to be an ADMIN of the organization or at
+    least a TEAM_MANAGER in the project(s) involved.
+
+    \b
+    !!! Examples
+        ```
+        kili project member add \\
+            john.doe@test.com \\
+            --role REVIEWER \\
+            --project-id <project_id>
+        ```
+        ```
+        kili project member add \\
+            path/to/members.csv \\
+            --project-id <project_id>
+        ```
+        ```
+        kili project member add \\
+            another_project_id \\
+            --project-id <project_id>
+        ```
+    """
+    kili = Kili(api_key=api_key, api_endpoint=endpoint)
+    emails = []
+    roles = []
+    for input_ in inputs:
+        if os.path.exists(input_):
+            read_csv_user(input_, emails, roles)
+        elif re.search(REGEX_EMAIL, input_):
+            emails.append(input_)
+        else:
+            extract_project_user(kili, input_, emails, roles)
+
+    if len(roles) == 0:
+        if role is None:
+            role = 'LABELER'
+        roles = [role] * len(emails)
+    elif len(roles) == len(emails):
+        if role is not None:
+            raise ValueError(
+                '--role cannot be used if the argument passed is '
+                'a path to a csv file with roles or a Kili project_id')
+    else:
+        raise ValueError(
+            'number of roles identified does not match the number of email adresses'
+            ', check the csv file')
+
```

Comment:
I am not convinced that it is a lot simpler (because we have to accept the multiple arguments, and I will have to handle every case, but I will try to see what I would look like)